web/app/view/Users.js

```diff
@@ -81,6 +81,20 @@ Ext.define('Traccar.view.Users', {
         }, {
             text: Strings.userAdmin,
             dataIndex: 'admin'
+        }, {
+            text: Strings.userDisabled,
+            dataIndex: 'disabled'
+        }, {
+            text: Strings.userExpirationTime,
+            dataIndex: 'expirationTime',
+            hidden: true,
+            renderer: function (value) {
+                if (Traccar.app.getPreference('twelveHourFormat', false)) {
```

Comment:
Would it be possible to share this logic somewhere. For example in AttributeFormatter.